src/ios/VLCPlugin.m

```diff
@@ -630,6 +656,24 @@ - (NSString*) vlc_convertAudioStateToString:(int) state{
 
 }
 
+- (void)vlc_resume {
+    if (self.activeMediaType == VLCPluginMediaTypeOnDemand) {
+        if (self.activePK) {
+            NSString *js = [NSString stringWithFormat:@"window.localStorage.getItem('audio-status-%@')", self.activePK];
```

Comment:
Similar concern as above -- someone else's javascript implementation may not utilize this local storage key...